### PR TITLE
Tests for dutch auction

### PR DESCRIPTION
**Pull request by OrysiaDrabych**
This change is [Reviewable](https://reviewable.io/reviews/openprocurement/robot_tests/624)

### CODE REVIEW

File: op_robot_tests/tests_files/insider_full.robot [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+  # ...      AND
+  # ...      Переключитись на учасника  ${provider}
+  # ...      AND
+  # ...      Page should contain  Аукціон завершився
+  # ...      AND
+  # ...      Переключитись на учасника  ${provider1}
+  # ...      AND
+  # ...      Page should contain  Аукціон завершився
+  # ...      AND
+  # ...      Close browser
```
Comment (ivanka12) [inline op_robot_tests/tests_files/insider_full.robot]: Навіщо зберігати закоментовані рядки?

File: op_robot_tests/tests_files/insider_full.robot [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+${sealedbid_amount}  xpath=(//div[contains(concat(' ', normalize-space(@class), ' '), ' sealedbid-winner ')]//span[@class='label-price ng-binding'])
+
+
+*** Test Cases ***
+Можливість знайти закупівлю по ідентифікатору
+  [Tags]   ${USERS.users['${viewer}'].broker}: Пошук тендера
+  ...      ${USERS.users['${viewer}'].broker}
+  ...      find_tender
+  Завантажити дані про тендер
+  Можливість знайти тендер по ідентифікатору для усіх користувачів
```
Comment (ivanka12) [inline op_robot_tests/tests_files/insider_full.robot]: якщо всі користувачі, то чому тільки viewer?

File: op_robot_tests/tests_files/insider_full.robot [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+##############################################################################################
+#             AUCTION
+##############################################################################################
+
+Відображення дати початку аукціону
+  [Tags]   ${USERS.users['${viewer}'].broker}: Відображення основних даних аукціону
+  ...      viewer
+  ...      ${USERS.users['${viewer}'].broker}
+  ...      tender_view
+  Дочекатись дати початку періоду аукціону  ${viewer}  ${TENDER['TENDER_UAID']}
```
Comment (kosaniak) [inline op_robot_tests/tests_files/insider_full.robot]: я би тут і в тк нижче додала
 +  [Setup]  Дочекатись синхронізації з майданчиком  ${viewer}
 +  [Teardown]  Оновити LAST_MODIFICATION_DATE

**Comment by OrysiaDrabych** [comment]
Review status: 0 of 10 files reviewed at latest revision, 7 unresolved discussions.

---

*[op_robot_tests/tests_files/insider_full.robot, line 20 at r6](https://reviewable.io:443/reviews/openprocurement/robot_tests/624#-KvSHqRElEEoc_2RZ4LS-r6-20:-KvbFZRsB-cZCEKyhvKl:b-896fix) ([raw file](https://github.com/openprocurement/robot_tests/blob/9c1e6d218b8673b426040972720382c571bfb3af/op_robot_tests/tests_files/insider_full.robot#L20)):*
<details><summary><i>Previously, ivanka12 wrote…</i></summary><blockquote>

якщо всі користувачі, то чому тільки viewer?
</blockquote></details>

Done.

---

*[op_robot_tests/tests_files/insider_full.robot, line 31 at r6](https://reviewable.io:443/reviews/openprocurement/robot_tests/624#-KvSHqRElEEoc_2RZ4LS-r6-31:-KvbH0ZEIdnAjWgOZfVE:b-896fix) ([raw file](https://github.com/openprocurement/robot_tests/blob/9c1e6d218b8673b426040972720382c571bfb3af/op_robot_tests/tests_files/insider_full.robot#L31)):*
<details><summary><i>Previously, kosaniak wrote…</i></summary><blockquote>

я би тут і в тк нижче додала
 +  [Setup]  Дочекатись синхронізації з майданчиком  ${viewer}
 +  [Teardown]  Оновити LAST_MODIFICATION_DATE
</blockquote></details>

Done.

---

*[op_robot_tests/tests_files/insider_full.robot, line 125 at r6](https://reviewable.io:443/reviews/openprocurement/robot_tests/624#-KvSHqRElEEoc_2RZ4LS-r6-125:-KvbH3KtN94d-F9H7Szx:b-896fix) ([raw file](https://github.com/openprocurement/robot_tests/blob/9c1e6d218b8673b426040972720382c571bfb3af/op_robot_tests/tests_files/insider_full.robot#L125)):*
<details><summary><i>Previously, kosaniak wrote…</i></summary><blockquote>

я би додала [Teardown]  Оновити LAST_MODIFICATION_DATE
</blockquote></details>

Done.

---

*[op_robot_tests/tests_files/insider_full.robot, line 180 at r6](https://reviewable.io:443/reviews/openprocurement/robot_tests/624#-KvSHqRElEEoc_2RZ4LS-r6-180:-KvbH5WtIHLPtIpZFmPf:b-896fix) ([raw file](https://github.com/openprocurement/robot_tests/blob/9c1e6d218b8673b426040972720382c571bfb3af/op_robot_tests/tests_files/insider_full.robot#L180)):*
<details><summary><i>Previously, kosaniak wrote…</i></summary><blockquote>

краще було би використати wait_and_write_to_console
</blockquote></details>

Done.

---

*[op_robot_tests/tests_files/insider_full.robot, line 237 at r6](https://reviewable.io:443/reviews/openprocurement/robot_tests/624#-KvSHqRElEEoc_2RZ4LS-r6-237:-KvbGwqcSVDfDIu5l47l:b-896fix) ([raw file](https://github.com/openprocurement/robot_tests/blob/9c1e6d218b8673b426040972720382c571bfb3af/op_robot_tests/tests_files/insider_full.robot#L237)):*
<details><summary><i>Previously, ivanka12 wrote…</i></summary><blockquote>

Навіщо зберігати закоментовані рядки?
</blockquote></details>

Done.

---

*[op_robot_tests/tests_files/insider_full.robot, line 276 at r6](https://reviewable.io:443/reviews/openprocurement/robot_tests/624#-KvSHqRElEEoc_2RZ4LS-r6-276:-KvbH8GtoJvsR9d14qE7:b-896fix) ([raw file](https://github.com/openprocurement/robot_tests/blob/9c1e6d218b8673b426040972720382c571bfb3af/op_robot_tests/tests_files/insider_full.robot#L276)):*
<details><summary><i>Previously, kosaniak wrote…</i></summary><blockquote>

для чого нам потрібна далі ця змінна?
</blockquote></details>

Done.

---

*[op_robot_tests/tests_files/keywords.robot, line 883 at r6](https://reviewable.io:443/reviews/openprocurement/robot_tests/624#-KvSHqRDgu-_i_k5NGgU-r6-883:-KvbHBFL037gdPOG4rf_:b-896fix) ([raw file](https://github.com/openprocurement/robot_tests/blob/9c1e6d218b8673b426040972720382c571bfb3af/op_robot_tests/tests_files/keywords.robot#L883)):*
<details><summary><i>Previously, kosaniak wrote…</i></summary><blockquote>

для проходження авардів майданчики зменшують акселератор і їм прийдеться чекати на зміну статусу не 5 хв, а 30 чи більше. 
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/robot_tests/624)*

**Pull request merged**